{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/globals.py

```diff
@@ -0,0 +1,144 @@
+####################
+# Project metadata #
+####################
+
+PROJECT_NAME = '{{cookiecutter.package_name}}'
+CLUSTER_PROJECT = 'proj_cost_effect'
+
+CLUSTER_QUEUE = 'all.q'
+MAKE_ARTIFACT_MEM = '3G'
+MAKE_ARTIFACT_CPU = '1'
+MAKE_ARTIFACT_RUNTIME = '3:00:00'
+
+LOCATIONS = [
+    # TODO - project locations here
+]
+
+
+#############
+# Data Keys #
+#############
+
+METADATA_LOCATIONS = 'metadata.locations'
+
+POPULATION_STRUCTURE = 'population.structure'
+POPULATION_AGE_BINS = 'population.age_bins'
+POPULATION_DEMOGRAPHY = 'population.demographic_dimensions'
+
+ALL_CAUSE_CSMR = 'cause.all_causes.cause_specific_mortality_rate'
+
+# TODO - sample disease keys
+DIARRHEA_CAUSE_SPECIFIC_MORTALITY_RATE = 'cause.diarrheal_diseases.cause_specific_mortality_rate'
+DIARRHEA_PREVALENCE = 'cause.diarrheal_diseases.prevalence'
+DIARRHEA_INCIDENCE_RATE = 'cause.diarrheal_diseases.incidence_rate'
+DIARRHEA_REMISSION_RATE = 'cause.diarrheal_diseases.remission_rate'
+DIARRHEA_EXCESS_MORTALITY_RATE = 'cause.diarrheal_diseases.excess_mortality_rate'
+DIARRHEA_DISABILITY_WEIGHT = 'cause.diarrheal_diseases.disability_weight'
+DIARRHEA_RESTRICTIONS = 'cause.diarrheal_diseases.restrictions'
+
+
+### ------------- Reviewers: what, if any, of the following should be part of the template ------------- 
+'''
+###########################
+# Disease Model variables #
+###########################
+
+# TODO - sample states and transitions
```

Comment:
I don't know if I'm totally ready to commit to this yet.  It's clunkier even then the disease model api: https://github.com/ihmeuw/vivarium_public_health/blob/master/src/vivarium_public_health/disease/models.py
which makes me super unhappy.  It's useful though.

Reply:
Keep this for now.